eth_account/account.py

```diff
@@ -192,7 +196,11 @@ def privateKeyToAccount(self, private_key):
             # They correspond to the same-named methods in Account.*
             # but without the private key argument
         '''
-        key_bytes = HexBytes(private_key)
+        if isinstance(private_key, self._keys.PrivateKey):
+            key_bytes = private_key.to_bytes()
+        else:
+            key_bytes = HexBytes(private_key)
```

Comment:
I suppose this could follow the pattern you used elsewhere, of using the `PrivateKey` directly as `key_obj` if `private_key` is already the right type.

Reply:
I added a new private method to add some reusability since I was repeating the pattern of generating a `eth_keys.datatypes.PrivateKey`. Open to suggestion on the naming / docs. This will also allow us to preserve the `ValueError` method as it would be difficult to follow the same pattern in this snippet while preserving the ValueError check (without having to have some repetitive type-checking code).

https://github.com/ethereum/eth-account/pull/30/commits/b501da3a3ba80fb9e62ce58b20575ca91f2548e7#diff-a9c6ca5d3c0ddc6c626bcee1142503c9R448

Reply:
Looks great, nothing to add!